src/repository/comments.py

```diff
@@ -1,10 +1,29 @@
 from datetime import datetime
+from typing import List
 
+from sqlalchemy import Select, asc
 from sqlalchemy.orm import Session
 
 from src.database.models import User, PhotoComment
 from src.schemas import CommentSchema
 
+async def get_comment(comment_id:int, db:Session) -> PhotoComment | None:
+    """
+    The get_comment function returns a comment object from the database.
+        
+    :param comment_id:int: Find the comment in the database
+    :type comment_id:int
+    :param db:Session: Connect to the database
+    :type db: Session
+    :return: A photocomment or none
+    :rtype: PhotoComment | None
+    """
+    stmt = Select(PhotoComment).filter_by(id=comment_id)
```

Comment:
Rename 'stmt' to something understandable.

Reply:
done